### PR TITLE
Force data only mode on XLSX format

### PR DESCRIPTION
**Pull request by sarendsen**
**Problem**

Solves https://github.com/django-import-export/django-import-export/issues/1003

**Solution**

Force formula to value on import.

### CODE REVIEW

**Comment by andrewgy8** [comment]
Hey thanks for the contrib! I'm embarrassed to say, but I think this slipped through the cracks. Could we create a test for this change?

File: import_export/formats/base_formats.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def create_dataset(self, in_stream):
         """
         from io import BytesIO
         import openpyxl
-        xlsx_book = openpyxl.load_workbook(BytesIO(in_stream), read_only=True)
+        xlsx_book = openpyxl.load_workbook(BytesIO(in_stream), read_only=True, data_only=True)
```
Comment (andrewgy8) [inline import_export/formats/base_formats.py:176]: Could yo explain the ramifications of this change? Would it have any effect on the users experience?

Comment (sarendsen) [reply import_export/formats/base_formats.py:176]: Instead of getting a formula on import now you get the value of that formula.

**Comment by andrewgy8** [comment]
Thanks for the contribution. I'm going to convert this to a draft since it is lacking test coverage and documentation.

**Comment by matthewhegarty** [comment]
Have manually tested this and added a new unit test.

**Review by manelclos — COMMENTED** [review]
I'd say that we need another test where we have a formula, like A1: 2, B1: 3 and C1: =A1*B1, would result in importing a '6' value.

**Comment by matthewhegarty** [comment]
Have updated the existing `test_import()` to check the results read from the test xlsx file.
It includes a new column (price), which contains a formula:

```
=abs(-4)
```

The test now proves that the result of this formula (4) is imported instead of the formula itself.

**Review by manelclos — APPROVED** [review]
Great!

**Pull request merged**